kie-wb-smoke-tests/src/test/java/org/kie/smoke/wb/selenium/ui/LoginIntegrationTest.java

```diff
@@ -40,4 +42,16 @@ public void loginAndLogout() throws Exception {
         login = home.logout();
         assertTrue(login.isDisplayed());
     }
+
+    @Test
+    public void traverseAllPerspectives() {
+        LoginPage login = pof.createLoginPage();
+        HomePerspective home = login.loginAs(KIE_USER, KIE_PASS);
+        PrimaryNavbar navbar = home.getNavbar();
+        for (Persp p : Persp.getAllPerspectives()) {
+            AbstractPerspective persp = navbar.navigateTo(p);
+            assertTrue("Perspective " + p.getName() + " should be loaded", persp.isDisplayed());
+        }
+        navbar.logout();
+    }
```

Comment:
Just a thought -- could we add a separate test for each perspective? For example using the Parametrized runner. I think it might useful to know how many perspectives failed to load, not just seeing the first one that failed to load. WDYT?